staging/src/kubevirt.io/client-go/apis/core/v1/schema.go

```diff
@@ -547,8 +547,26 @@ type FilesystemVirtiofs struct{}
 // +k8s:openapi-gen=true
 type GPU struct {
 	// Name of the GPU device as exposed by a device plugin
-	Name       string `json:"name"`
-	DeviceName string `json:"deviceName"`
+	Name              string       `json:"name"`
+	DeviceName        string       `json:"deviceName"`
+	VirtualGPUOptions *VGPUOptions `json:"virtualGPUOptions,omitempty"`
+}
+
+//
+// +k8s:openapi-gen=true
+type VGPUOptions struct {
+	Display *VGPUDisplayOptions `json:"display,omitempty"`
+}
+
+//
+// +k8s:openapi-gen=true
+type VGPUDisplayOptions struct {
+	// Enabled determines if a display addapter backed by a vGPU should be enabled or disabled on the guest.
+	// Defaults to true.
+	// +optional
+	Enabled *bool `json:"enabled,omitempty"`
+	// Enables a boot framebuffer, until the guest OS loads a real GPU driver
+	RamFB *FeatureState `json:"ramFB,omitempty"`
```

Comment:
Is this enabled by default? I could not tell from the comment which will also land in the API docs.

Reply:
Yes, this will be enabled by default. 
This PR adds a virtual display for each vGPU by default. RamFB will also be enabled by default for each of the vGPU displays.
The user will be able to turn off either RamFB separately:
```
gpus:
      - deviceName: nvidia.com/GRID_T4-2Q
        name: myvgpu
        virtualGPUOptions:
          display:
            enabled: true
            ramFB:
                enabled: false
```


or all the vGPU display settings:
```
   gpus:
      - deviceName: nvidia.com/GRID_T4-2Q
        name: myvgpu
        virtualGPUOptions:
          display:
            enabled: false
```

Reply:
I've adjusted the API docs